src/web/client/dal/remoteSaveProvider.ts

```diff
@@ -157,7 +158,11 @@ async function saveDataToDataverse(
                 saveCallParameters.requestUrl,
                 entityName,
                 httpMethod.PATCH,
-                fileExtensionType
+                fileExtensionType,
+                undefined,
+                0,
+                undefined,
+                telemetryEventNames.WEB_EXTENSION_SAVE_FILE_TRIGGERED
```

Comment:
I would suggest adding a new telemetry event here - one specific to dataverse save call triggered

Reply:
done